build.gradle

```diff
@@ -131,6 +131,11 @@ def versions = [
 ]
 
 dependencyManagement {
+  dependencies {
```

Comment:
hehe
https://github.com/hmcts/cmc-claim-store/blob/master/build.gradle#L88

Reply:
yea. added in first stab in send letter service. over here did "select all and delete" :(